ansible/roles/maven/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Creating directory for maven download
+  file:
+    path: /maven
+    state: directory
+    owner: '{{work_user}}'
+  become: true
+
+- name: Downloading maven for Dspace backend
+  get_url:
+    url: "{{ maven_download_url }}"
+    dest: "/maven/apache-maven-3.8.4-bin.tar.gz"
+ 
+- name: Expand dspace repository just downloaded.
+  unarchive:
+    src: "/maven/apache-maven-3.8.4-bin.tar.gz"
+    dest: "/maven"
+    remote_src: true  
+  
+
+- name: mv maven file
+  copy:
+    src: /maven/apache-maven-3.8.4/
```

Comment:
there is an existing ansible variable for the maven-version `maven_version`

Reply:
made changes